modules/appnexusBidAdapter.md

```diff
@@ -89,7 +89,16 @@ var adUnits = [
      mediaTypes: {
        video: {
          playerSize: [[300, 250]],
-         context: 'outstream'
+         context: 'outstream',
+         // certain ORTB 2.5 video values can be read from the mediatypes object; below are examples of supported params
+         // to note - appnexus supports additional values for our system that are not part of the ORTB spec.  If you want
+         // to use these values, they will have to be declared in the bids[].params.video object instead using the appnexus syntax.
+         minduration: 1,
+         maxduration: 60,
+         skip: 1,   // 1 - true, 0 - false
```

Comment:
In this example you have skip and playback method double set, is this intentional?

Reply:
The idea is that you can set both - but the ones in the bid.params will taken precedence due to unique values we have to support.

I'll add another comment to better clarify this point.